docusaurus/src/theme/Heading/styles.scss

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// overwriting docusaurus theme classic styles
+h1.docTitle_node_modules-\@docusaurus-theme-classic-lib-next-theme-DocItem- {
```

Comment:
hey, do you think we can find another way to change that? this `node_modules` thing makes me feel like its auto generated. maybe once the heading is swizzled we can manually add another one?

what exactly is this selector selecting? if its the title of the page, in stg that is different: 

![image](https://user-images.githubusercontent.com/18744505/120042290-2ccacd80-bfe0-11eb-9d13-502e54f0fa72.png)

Reply:
Yeah, it was supposed to select the page title, I didn't realise it was randomly generated 🤦  I'll see if I can select it differently or swizzle it if necessary